src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyAmount = 0;
+        int buyAmount = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String textLines = bufferedReader.readLine();
+
+            while (textLines != null) {
+                String[] lineInfo = textLines.split(",");
+                if (lineInfo[0].equals(SUPPLY)) {
```

Comment:
let's make divider and indexes, which represent operation and amount, constant variables